applications/nrf5340_audio/src/bluetooth/bt_stream/CMakeLists.txt

```diff
@@ -5,11 +5,16 @@
 #
 
 zephyr_library_include_directories(
-    broadcast
-    unicast
+	broadcast
+	unicast
 )
 
 if (CONFIG_TRANSPORT_BIS)
 	target_sources(app PRIVATE
 			${CMAKE_CURRENT_SOURCE_DIR}/broadcast/broadcast_source.c)
 endif()
+
+if (CONFIG_TRANSPORT_CIS)
```

Comment:
As we talked about: We should/could transition into using host configs for places like these.
Here CONFIG_BT_BAP_UNICAST_SERVER could be the best to use